### DISCUSSION TITLE
2023-11-07

### DISCUSSION THREAD

**Discussion by hanseulhee** [10-rolling/rollingPolling#38] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] 돈독한 회식
- [x] theme 구조, img & icon 파일 명 변경, 컴포넌트는 단일 태그, global font size 변경, button cursor
- [x] 각자 조사 해온 거 발표
- [x] 기능 정리해서 나누기
- [x] 다음 개발 역할 분담

### 📝 회의내용

- 컨디션 어떠세요? (전 너무 힘들어요)
- 오늘의 포부 이야기하기
- 각자 오늘 할 개발 이야기하기
- 궁금한 거, 공유할 거 이야기하기

- 데이터 채워놨어요
<img width="35%" alt="스크린샷 2023-11-07 오전 3 48 49" src="https://github.com/10-rolling/rollingPolling/assets/63100352/c2fe5dea-e96a-4a97-b59a-b1dadc411c23">

### 💪🏻 오늘 할일

- [x] 각자 맡은 거 개발하기
- [x] 멘토링때 궁금한 거 각자 정리해보기
- [x] 해장하기

### 🤔 특이사항

<img src="https://github.com/10-rolling/rollingPolling/assets/63100352/132daf64-9a4d-4cbb-a9ad-4e4d84f54465" width="40%" />
<img src="https://github.com/10-rolling/rollingPolling/assets/63100352/a875a41b-4774-46c3-960f-4233de311d77" width="34%" />

아 ... 어질어질 

저희 언제쯤 조금 먹어요? 완전 롤링롤링 하네요 .....

**Comment by ngsy** [comment]
어우 다들 너무 잘드셔요🫠

**Comment by miniposi** [comment]
와 데이터 언제 채워 두셨어요 👀👏
